endpoint/platform/legato-target/build.mk

```diff
@@ -5,4 +5,4 @@
 # "LICENSE" at the root of this distribution.
 
 platform-build-command = \
-	cd endpoint && leaf shell -c "mkapp -v -t wp77xx $(LEGATO_FLAGS) endpoint.adef"
+	cd endpoint && leaf shell -c "mkapp -v -t $(EP_TARGET) $(LEGATO_FLAGS) endpoint.adef"
```

Comment:
The command is so different from `endpoint/platform/simulator/build.mk`.
Including the `leaf shell` and `-i ${LEGATO_ROOT}/interfaces/positioning`.
Could you explain more?